PropertyChanged.Fody/PropertyWeaver.cs

```diff
@@ -72,6 +72,10 @@ void InjectAtIndex(int index)
             }
         }
 
+        var setMethod = propertyData.PropertyDefinition.SetMethod;
+        if (setMethod.GetNonAbstractBaseMethod(out _))
+            return;
```

Comment:
You need to check if the base method is actually called. If it's not, you still need to insert the event invoker.

I added the following:
 - `public virtual int Property4 { get; set; }` in `BaseClass`
 - `public override int Property4 { get; set; }` in `DerivedClass`

And the following test fails:

```C#
[Fact]
public void DerivedPropertyNotCallingBasePropertyShouldRaiseEvents()
{
    var instance = testResult.GetInstance("DerivedClass");
    instance.Property4 = 42;

    var expected = new[] { "Property4" };

    Assert.Equal(expected, (IList<string>)instance.ChangedEvents);
}
```

Reply:
I would not consider this an error. 
If you override a property without calling the base classes methods, you intend to overrude the original behavior with something different - so here you give up property change notification.

Reply:
And yes, more tests should be added. Maybe @DaleStan can jump in and port his tests, if we agree on this approach.

Reply:
@ltrzesniewski now handling this use case, and added more tests.